xbmc/guilib/GUIControl.h

```diff
@@ -219,6 +219,7 @@ class CGUIControl
   virtual void SetInvalid() { m_bInvalidated = true; };
   virtual void SetPulseOnSelect(bool pulse) { m_pulseOnSelect = pulse; };
   virtual std::string GetDescription() const { return ""; };
+  virtual std::string GetDescriptionByIndex(int index) const { return ""; };
```

Comment:
why did you changed the method name? it's fine if you have different method signatures for one method name.